op-deployer/pkg/deployer/state/state.go

```diff
@@ -40,6 +37,9 @@ type State struct {
 
 	// Chains contains data about L2 chain deployments.
 	Chains []*ChainState `json:"opChainDeployments"`
+
+	// L1StateDump contains the complete L1 state dump of the deployment.
+	L1StateDump *GzipData[foundry.ForgeAllocs] `json:"stateDump"`
```

Comment:
Might be nice to have this, so that it's more explicit in the `state.json` file. 

```suggestion
	L1StateDump *GzipData[foundry.ForgeAllocs] `json:"l1StateDump"`
```